patch-package
--- a/node_modules/@ecl/ec-component-navigation-menu/ec-component-navigation-menu.twig
+++ b/node_modules/@ecl/ec-component-navigation-menu/ec-component-navigation-menu.twig
@@ -48,7 +48,7 @@
 
 <nav class="{{ _css_class }}" aria-label="{{ _menu_aria_label }}"{{ _extra_attributes|raw }}>
   <div class="ecl-container">
-    <button class="ecl-navigation-menu__toggle ecl-navigation-menu__hamburger ecl-navigation-menu__hamburger--squeeze" aria-controls="nav-menu-expandable-root" aria-expanded="false">
+    <button class="ecl-navigation-menu__toggle ecl-navigation-menu__hamburger ecl-navigation-menu__hamburger--squeeze" aria-label="menu" aria-controls="nav-menu-expandable-root" aria-expanded="false">
       <span class="ecl-navigation-menu__hamburger-box">
         <span class="ecl-navigation-menu__hamburger-inner"></span>
       </span>
