patch-package 
--- a/node_modules/@ecl/generic-component-logo/generic-component-logo.twig
+++ b/node_modules/@ecl/generic-component-logo/generic-component-logo.twig
@@ -49,7 +49,7 @@
 {# Print the result #}
 
 {% if _href is not empty %}
-  <a href="{{ _href }}" class="{{ _css_class }}" title="{{ _title }}"{{ _extra_attributes|raw }}><span class="ecl-u-sr-only">{{ _title }}</span></a>
+  <a href="{{ _href }}" class="{{ _css_class }}" title="{{ _title }}"{{ _extra_attributes|raw }}><span class="ecl-u-sr-only"></span></a>
 {% else %}
   <span class="{{ _css_class }}" role="img" aria-label="{{ _title }}"{{ _extra_attributes|raw }}></span>
 {% endif %}
